src/main/java/com/opentable/service/ServiceInfo.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.opentable.service;
+
+/**
+ * The {@link ServiceInfo} class is intended for you to declare as a Spring Bean in your application.
+ * It can be consumed by our components in order to create things like metric namespaces given information about your
+ * service.
+ */
+public class ServiceInfo {
```

Comment:
Another idea: you could implement a `ServiceInfo` interface.  That way each individual piece of information is named.  e.g. contrast

```java
new ServiceInfo("my-name", new Version(1.0.0.), someMetadata);
```

```java
interface ServiceInfo {
    String getServiceName();
    Version getServiceVersion();
    Map<String, Object> getMetadata();
}
class FooServiceInfo implements ServiceInfo {
    ...
}
```